.gitignore

```diff
@@ -3,6 +3,6 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly included to ignore virtual environment directories. However, ensure that this is the correct directory name for your virtual environment. If your virtual environment is named differently, adjust this entry accordingly.